juniper_official/lldp/lldp-session-state.rule

```diff
@@ -23,62 +23,69 @@ healthbot {
              */
             sensor lldp-sensor {
                 synopsis "LLDP open-config sensor definition";
-                description "/lldp open-config sensor to collect telemetry data from network device";
-                open-config {
-                    sensor-name /lldp/;
-                    frequency 60s;
+                description "lldp netconf sensor to collect data from network device";
+                iAgent {
+                    file LldpNeighborInfoTable.yml;
+                    table LldpNeighborInfoTable;
+                    frequency 180s;
                 }
             }
             /*
              * Fields defined using sensor path. Map the longer sensor names
              * to the shorter field names used in the rules.
              */
-            field lldp-enabled {
+            field remote-system-name {
                 sensor lldp-sensor {
-                    path /lldp/interfaces/interface/state/enabled;
+                    path remote-system-name;
```

Comment:
Can one interface have multiple remote-system?

Reply:
Thanks Vijay for review. No, there will be one remote-system per interface